packages/components/organisms/f-checkout/test/specs/component/f-checkout-collection.component.spec.js

```diff
@@ -3,9 +3,7 @@ import CheckoutComponent from '../../../test-utils/component-objects/f-checkout.
 
 describe('f-checkout "collection" component tests', () => {
     before(() => {
-        browser.url('?path=/story/components-organisms--checkout-component');
-        CheckoutComponent.changeCheckoutMethod('collection');
-        browser.switchToFrame(0);
+        browser.url('iframe.html?id=components-organisms--checkout-component&knob-Checkout%20Url=%2Fcheckout-collection.json&knob-Available%20Fulfilment%20Url=%2Fcheckout-available-fulfilment.json&knob-Create%20Guest%20Url=%2Fcreate-guest.json&knob-Auth%20token=a&knob-Login%20Url=%2Flogin&viewMode=story');
```

Comment:
Note: This is a temp url as this will be changing in the next 2 weeks after Lizzie has made her changes.